databricks/koalas/config.py

```diff
@@ -220,6 +220,17 @@ def validate(self, v: Any) -> None:
             "'plotting.sample_ratio' should be 1.0 >= value >= 0.0.",
         ),
     ),
+    Option(
+        key="plotting.backend",
+        doc=(
+            "Backend to use for plotting. Default is matplotlib. "
+            "Supports any package that has a top-level `.plot` method. "
+            "Some options are: [matplotlib, plotly, pandas_bokeh, pandas_altair]."
+        ),
+        default="matplotlib",
+        types=str,
+        check_func=(lambda v: isinstance(v, str), "'plotting.backend' should b a string",),
```

Comment:
This check is done by `types=str`.

How about:

```py
        check_func=(
            lambda v: v in ("matplotlib", "plotly", "pandas_bokeh", "pandas_altair"),
            "'plotting.backend' should be one of 'matplotlib', 'plotly', 'pandas_bokeh', 'pandas_altair'",
        ),
```

Reply:
This will force the user to choose from the given options. Any library or their own custom library that supports a top-level `.plot` function will not be usable.

Reply:
ah, so any libraries which satisfy the convention, e.g. have `plot` function, etc., can work as backend?

Reply:
if yes above, shall we remove the check?

Reply:
Yes

Reply:
Could you remove the check here?